apisix/discovery/kubernetes/init.lua

```diff
@@ -520,6 +520,12 @@ end
 
 
 function _M.init_worker()
+    if not support_process then
+        core.log.error("kubernetes discovery not support in subsystem: "
+                       .. ngx.config.subsystem
```

Comment:
We can use ',' here

Reply:
@spacewander do you mean something like below? `.. ngx.config.subsystem .. ", "`

```lua
if not support_process then
    core.log.error("kubernetes discovery not support in subsystem: "
                   .. ngx.config.subsystem .. ", "
                   .. "please check if your openresty version >= 1.19.9.1 or not")
    return
end
```

Reply:
Err. What I mean is that we can use ',' instead of '..'.

Reply:
ok, i see. I will change it later.